cagov/ui-claim-tracker
# Settings  Add program/extension type mappings for PEUC, PEUX, PEUY program types

## Ticket

Resolves #413

## Changes

- Add logic and English language text for program extension types: PEUC, PEUX, PEUY
- Fix regression: move `Claim Status Tracker` back to the top of the storybook sidebar

## Context

We received information that these program types will be sent _in addition_ to the non-pandemic versions of these program types.

## Testing

1. Check out this branch
2. Run `yarn storybook`
3. Use the controls to view the PEUC, PEUX, and PEUY program types and verify against the description of #413
